src/Components/Display-card/DisplayCard.jsx

```diff
@@ -1,28 +1,27 @@
 import "../../css/main.css";
 import "./displaycard.css";
+import { useNotes } from "../../Context";
 
-export const DisplayCard = ({color}) => {
+export const DisplayCard = ({item}) => {
+    const { _id, pinned, title, description, tag, priority, selectedBackgroundColor } = item;
+    const { archiveNote, deleteNote } = useNotes();
     return(
         <>
-        <div className="display-card flex flex-column pd-1" style={{backgroundColor: color}}>
-            <button className="btn-transparent btn-pinned"><span class="material-icons btn-color">push_pin</span></button>
+        <div className="display-card flex flex-column pd-1" style={{backgroundColor: selectedBackgroundColor}} key={_id}>
```

Comment:
Suggestions :-   Try to use class instead of using inline style to simplify code

Reply:
I have no other option since I am setting the background color as a prop value